src/App/App.tsx

```diff
@@ -1,12 +1,17 @@
 import type { FC } from "react";
 import { BrowserRouter, Route, Routes } from "react-router";
-import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
+import { ApolloClient, HttpLink, InMemoryCache } from "@apollo/client";
+import { Defer20220824Handler } from "@apollo/client/incremental";
+import { ApolloProvider } from "@apollo/client/react";
 import { AppLayout } from "./AppLayout";
 import { BooksRoute } from "./BooksRoute";
 import { BookRoute } from "./BookRoute";
 
 const apolloClient = new ApolloClient({
-  uri: "http://localhost:4488/graphql", // https://github.com/eddeee888/graphql-server-template
+  link: new HttpLink({
+    uri: "http://localhost:4488/graphql", // https://github.com/eddeee888/graphql-server-template
+  }),
+  incrementalHandler: new Defer20220824Handler(),
```

Comment:
Handle defer, since our server counterpart supports it: https://github.com/eddeee888/graphql-server-template/pull/135